### ISSUE TITLE
Build failure on node 6.0.0

### ISSUE THREAD

**Issue by dreambottle** [atom/node-keytar#40]
My system is OS X
When I'm running `npm install keytar`, I get the following output:

```
> keytar@3.0.1 install /Users/nico/Workspace/Fractal/fractal-flasher-electron/node_modules/keytar
> node-gyp rebuild

  CXX(target) Release/obj.target/keytar/src/main.o
In file included from ../src/main.cc:1:
../node_modules/nan/nan.h:582:20: error: no type named 'GCEpilogueCallback' in
      'v8::Isolate'
      v8::Isolate::GCEpilogueCallback callback
      ~~~~~~~~~~~~~^
../node_modules/nan/nan.h:588:20: error: no type named 'GCEpilogueCallback' in
      'v8::Isolate'
      v8::Isolate::GCEpilogueCallback callback) {
      ~~~~~~~~~~~~~^
../node_modules/nan/nan.h:593:20: error: no type named 'GCPrologueCallback' in
      'v8::Isolate'
      v8::Isolate::GCPrologueCallback callback
      ~~~~~~~~~~~~~^
../node_modules/nan/nan.h:599:20: error: no type named 'GCPrologueCallback' in
      'v8::Isolate'
      v8::Isolate::GCPrologueCallback callback) {
      ~~~~~~~~~~~~~^
4 errors generated.
make: *** [Release/obj.target/keytar/src/main.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/nico/Workspace/Fractal/fractal-flasher-electron/node_modules/npm/node_modules/node-gyp/lib/build.js:276:23)
gyp ERR! stack     at emitTwo (events.js:106:13)
gyp ERR! stack     at ChildProcess.emit (events.js:191:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:204:12)
gyp ERR! System Darwin 15.4.0
gyp ERR! command "/usr/local/Cellar/node/6.0.0/bin/node" "/Users/nico/Workspace/Fractal/fractal-flasher-electron/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/nico/Workspace/Fractal/fractal-flasher-electron/node_modules/keytar
gyp ERR! node -v v6.0.0
gyp ERR! node-gyp -v v3.3.1
gyp ERR! not ok 
npm WARN fractal-tray@1.0.0 No description
npm WARN fractal-tray@1.0.0 No repository field.
npm ERR! Darwin 15.4.0
npm ERR! argv "/usr/local/Cellar/node/6.0.0/bin/node" "/Users/nico/Workspace/Fractal/fractal-flasher-electron/node_modules/.bin/npm" "install" "keytar"
npm ERR! node v6.0.0
npm ERR! npm  v3.8.8
npm ERR! code ELIFECYCLE

npm ERR! keytar@3.0.1 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the keytar@3.0.1 install script 'node-gyp rebuild'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the keytar package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs keytar
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls keytar
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/nico/Workspace/Fractal/fractal-flasher-electron/npm-debug.log
```

Luckily I have v.5.11.0 cached in brew and when I switch to 5.11.0
`brew switch node 5.11.0`
`npm install keytar`
everything goes fine, so this must be an issue with a new version of node

**Comment by as-cii** [comment]
Thanks for the report, @dreambottle and @50Wliu. We've recently switched to using `nan 2.0.0` because of a problem with building this package on a older version of Node. I think they have fixed the problem since then, and so I am going to open a PR that changes `nan` back to the latest version (this time I'll use a fixed version though).

**Comment by as-cii** [comment]
Published on version 3.0.2.
